Stdlib/Trait/Functor/Polymorphic.juvix

```diff
@@ -0,0 +1,23 @@
+module Stdlib.Trait.Functor.Polymorphic;
+
+import Stdlib.Data.Fixity open;
+import Stdlib.Data.Unit open;
+
+trait
+type Functor (f : Type -> Type) :=
+  mkFunctor {
+    syntax iterator map {init := 0; range := 1};
+    map : {A B : Type} -> (A -> B) -> f A -> f B
+  };
+
+open Functor public;
```

Comment:
The duplicate open Functor public can be removed?

Reply:
fixed